codecept.json

```diff
@@ -1,19 +1,11 @@
 {
   "output": "tests/output",
   "helpers": {
-    "WebDriverIO": {
-      "url": "{PMM_URL_HERE}",
+    "WebDriver": {
+      "url": "https://18.224.200.209/",
       "browser": "chrome",
-      "user": "pmm_qa",
-      "key": "{SAUCE_USER_KEY}",
-      "host": "ondemand.saucelabs.com",
       "windowSize": "maximize"
     },
-    "SauceHelper": {
```

Comment:
We need the helpers when running tests on our Jobs, for local setup you can create a copy of this and make changes